org/diceresearch/nebula/orchestrator.py

```diff
@@ -119,11 +126,17 @@ def goNextLevel(identifier):
 
     elif next_stage == 6:
         databasemanager.update_step(settings.results_table_name, settings.status, settings.done, identifier)
+        if REGISTRATION_TOKEN:
+            notification.send_firebase_notification(REGISTRATION_TOKEN, NOTIFICATION_TITLE, NOTIFICATION_BODY)
+
     elif next_stage == 7:
-        pass
+        databasemanager.delete_from_column(settings.results_table_name, settings.results_notificationtoken_column_name)
```

Comment:
Shouldn't that be done directly after the `notification.send_firebase_notification` call or is there a reason why we should wait? :thinking: